Adding section about political aspects of language to BLISS (#1443)
ihmeuw/vivarium_research: docs/source/model_design/general_reference_material/bliss/index.rst

   Include visual of 'Do's and Don'ts!

1.4 The Politics of Language
----------------------------
The language we choose to use is inherently political and we are aware of that fact. 

> This guide asserts that gender inclusive language is important and pushes for it's 

Reviewer: ```suggestion
This guide asserts that gender inclusive language is important and pushes for its
```

> inclusion. In doing so, we hope to advocate for gender minority inclusion 

Reviewer 2: ```suggestion
wider use in public health research and dissemination. In doing so, we hope to advocate for SGM inclusion
```
could use 'SGM' acronym since it was just defined in glossary above and we use it elsewhere in the guide

> in statisitcs and research. 

Reviewer: ```suggestion
in statistics and research.
```

> in statisitcs and research. 

Reviewer 2: ```suggestion
in this field. 
```
I made this redundant with my edit in the above line

empowerment might choose to use different language to advocate for other groups. For example, 
there are places where women's inclusion in clinical trials or datasets is not 
guaranteed and they are still pushing for anyone other than cis-gendered men to be included 
in research. This is important and vital work. Additionally, some groups want to push 
back on male centric lanaguage - such as defaulting to masculine pronouns for 
a non-gendered person. This is also a worthy cause. We recognize and respect that the intentional 
use of language to further a goal might lead some to different language choices in certain situtions. 

Some of the people in women's empowerment groups might feel that including gender 
inclusive language is antagonistic, detrimental, or mutually exclusive to women's empowerment. Our group 
disagrees with this proposed dichotomy. Having more diverse and precise language 
options can benefit both cis-women and gender minorities. There are wide variations 
in bodies among cis-women and moving towards better measurements will force 
researchers to define what they are hoping to quantify outside of just "male" or "female". 

> For example, considering "people with uteruses" instead of "women" when measuring uterine 

Reviewer 2: Could add a citation to Bauer 2022 paper for their research on this example

cancer rates would have helped better quantify incidence for all people. 

Our hope is that someday, sex and gender are better 

> measured and reported in data leading to less friction between groups. 

Reviewer 2: ```suggestion
measured and reported in data, leading to less friction between perspectives. 
```
